### PR TITLE
Add new atlas checklist

### PR DESCRIPTION
**Pull request by adamltyson**
This was everything I could think of. I'm sure there's more to add.

### CODE REVIEW

File: docs/source/community/developers/repositories/brainglobe-atlasapi/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# brainglobe-atlasapi
+
+## Adding a new atlas
+The details regarding packaging a new atlas can be found at 
+[adding a new atlas](/documentation/brainglobe-atlasapi/adding-a-new-atlas). Here we outline the steps that follow once 
+the new atlas pull request has been merged:
+
+1. Run the atlas script to generate the new atlas.
```
Comment (alessandrofelder) [inline docs/source/community/developers/repositories/brainglobe-atlasapi/index.md:8]:
```suggestion
1. Run the atlas script to generate the new atlas, and make sure it passes our validation.
```

**Review by alessandrofelder — APPROVED** [review]
Very nice. Small suggestion.

File: docs/source/community/developers/repositories/brainglobe-atlasapi/index.md (hunk 1 above)
Comment (adamltyson) [reply docs/source/community/developers/repositories/brainglobe-atlasapi/index.md:8]: I was going to add that, but that should be before the PR is merged right?

Comment (adamltyson) [reply docs/source/community/developers/repositories/brainglobe-atlasapi/index.md:8]: Or is it worth having a double check?

Comment (alessandrofelder) [reply docs/source/community/developers/repositories/brainglobe-atlasapi/index.md:8]: Hopefully this happens almost never, but I guess e.g. the dependency versions we generate the version we upload could be just different enough to mess with something? So I would lean towards double-checking.

We could reformulate to
```suggestion
1. Run the atlas script to generate the new atlas, and double-check it passes our validation.
```
to imply that it should be checked at PR review stage.

Comment (adamltyson) [reply docs/source/community/developers/repositories/brainglobe-atlasapi/index.md:8]: > the dependency versions we generate the version we upload could be just different enough to mess with something

I think I agree, but no idea what this means! 🤣

**Pull request merged**